deploy/terraform/modules/anydb_node/vm-anydb.tf

```
@@ -0,0 +1,140 @@
+#############################################################################
+# RESOURCES
+#############################################################################
+
+resource azurerm_network_interface "anydb" {
+  count               = local.enable_deployment ? length(local.dbnodes) : 0
+  name                = format("db%02d-%s-nic", (count.index + 1), local.anydb_sid)
+  location            = var.resource-group[0].location
+  resource_group_name = var.resource-group[0].name
+
+  ip_configuration {
+    primary                       = true
+    name                          = "${local.dbnodes[count.index].name}-db-nic-ip"
+    subnet_id                     = local.sub_db_exists ? data.azurerm_subnet.anydb[0].id : azurerm_subnet.anydb[0].id
+    private_ip_address            = local.sub_db_exists ? local.dbnodes[count.index].db_nic_ip : lookup(local.dbnodes[count.index], "db_nic_ip", false) != false ? local.dbnodes[count.index].db_nic_ip : cidrhost(local.sub_db_prefix, tonumber(count.index) + 10)
+    private_ip_address_allocation = "static"
+  }
+}
+
+# Section for Linux Virtual machine 
+resource azurerm_linux_virtual_machine "dbserver" {
+  count                        = local.enable_deployment ? ((upper(local.anydb_ostype) == "LINUX") ? length(local.dbnodes) : 0) : 0
+  name                         = format("db%02d-%s-vml", (count.index + 1), local.anydb_sid)
```

The naming convention has done once, don't need to do it again.
```suggestion
  name                         = local.dbnodes[count.index].name
```